superset/contrib/docker/Dockerfile

```diff
@@ -14,7 +14,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM python:3.6-jessie
+FROM node:12 AS static
+
+WORKDIR /home/superset
+
+COPY superset/assets superset/assets
+
+RUN cd superset/assets \
```

Comment:
> This also improves the usage of docker caching for this step that is really heavy.

if you want to improve it further you can do:
```
# install deps and cache them
COPY superset/assets/package*.json superset/assets
RUN npm ci

# build
COPY superset/assets superset/assets
RUN npm run build
```

It will use docker cache instead of downloading and installing dependencies most of the time.

Reply:
Thanks! Done [here](https://github.com/src-d/sourced-ui/pull/275/commits/a5d5b61ec0768bcd4b4b909b676d077c05ab8cf0).